src/Rector/Contrib/PHPUnit/SpecificMethodInternalTypeRector.php

```diff
@@ -0,0 +1,80 @@
+<?php declare(strict_types=1);
+
+namespace Rector\Rector\Contrib\PHPUnit;
+
+use PhpParser\Node;
+use PhpParser\Node\Arg;
+use PhpParser\Node\Expr\FuncCall;
+use PhpParser\Node\Expr\MethodCall;
+use PhpParser\Node\Identifier;
+use PhpParser\Node\Scalar\String_;
+use Rector\NodeAnalyzer\MethodCallAnalyzer;
+use Rector\Rector\AbstractRector;
+
+/**
+ * Before:
+ * - $this->assertTrue(is_string($anything));
+ *
+ * After:
+ * - $this->assertInternalType('string', $anything);
+ */
+final class SpecificMethodInternalTypeRector extends AbstractRector
+{
+    /**
+     * @var MethodCallAnalyzer
+     */
+    private $methodCallAnalyzer;
+
+    /**
+     * @var string|null
+     */
+    private $activeFuncCallName;
+
+    public function __construct(MethodCallAnalyzer $methodCallAnalyzer)
+    {
+        $this->methodCallAnalyzer = $methodCallAnalyzer;
+    }
+
+    public function isCandidate(Node $node): bool
+    {
+        $this->activeFuncCallName = null;
```

Comment:
@TomasVotruba What does this line do? I've kept it to follow standards, but couldn't figure out its function.

Reply:
This can be actually removed. It's probably left over before refactoring.